client/src/partials/stock/loss/loss.js

```diff
@@ -280,6 +280,24 @@ angular.module('bhima.controllers')
       });
     }
 
+    function getItemPrice(item, index) {
+      dependencies.itemPrice = {
+        query : {
+          tables : {
+            'stock' : { columns : ['purchase_order_uuid']},
+            'purchase_item' : { columns : ['quantity', 'unit_price']}
+          },
+          join : ['stock.inventory_uuid=purchase_item.inventory_uuid'],
+          where : ['stock.tracking_number=' + item.tracking_number]
+        }
+      };
+
+      validate.process(dependencies, ['itemPrice'])
```

Comment:
If this is called multiple times, you will want to use `validate.refresh(dependencies, array)`;